tests/infrastructure/taints-and-tolerations.go

```diff
@@ -22,23 +22,24 @@ package infrastructure
 import (
 	"context"
 	"fmt"
+	"strings"
 	"time"
 
-	"kubevirt.io/kubevirt/tests/framework/kubevirt"
-
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 
-	"kubevirt.io/kubevirt/tests/libnode"
-
 	k8sv1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/types"
-	"k8s.io/apimachinery/pkg/watch"
 	"kubevirt.io/client-go/kubecli"
 
 	"kubevirt.io/kubevirt/pkg/apimachinery/patch"
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/errors"
```

Comment:
Do you mean to use these `errors`?

Reply:
@akrejcir I've tested the below patch locally:

```diff
Index: tests/infrastructure/taints-and-tolerations.go
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/tests/infrastructure/taints-and-tolerations.go b/tests/infrastructure/taints-and-tolerations.go
--- a/tests/infrastructure/taints-and-tolerations.go	(revision 0d61ce55027199e205282dc402eab27784216641)
+++ b/tests/infrastructure/taints-and-tolerations.go	(date 1709631675257)
@@ -29,6 +29,7 @@
 	. "github.com/onsi/gomega"
 
 	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/fields"
 	"k8s.io/apimachinery/pkg/labels"
@@ -36,7 +37,6 @@
 	"kubevirt.io/client-go/kubecli"
 
 	"kubevirt.io/kubevirt/pkg/apimachinery/patch"
-	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/errors"
 	"kubevirt.io/kubevirt/tests/flags"
 	"kubevirt.io/kubevirt/tests/framework/kubevirt"
 	"kubevirt.io/kubevirt/tests/libnode"
Index: tests/infrastructure/BUILD.bazel
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/tests/infrastructure/BUILD.bazel b/tests/infrastructure/BUILD.bazel
--- a/tests/infrastructure/BUILD.bazel	(revision 0d61ce55027199e205282dc402eab27784216641)
+++ b/tests/infrastructure/BUILD.bazel	(date 1709631633068)
@@ -25,7 +25,6 @@
         "//pkg/util/tls:go_default_library",
         "//pkg/virt-config:go_default_library",
         "//pkg/virt-handler/node-labeller/util:go_default_library",
-        "//pkg/virt-launcher/virtwrap/errors:go_default_library",
         "//pkg/virt-operator/resource/generate/components:go_default_library",
         "//staging/src/kubevirt.io/api/core/v1:go_default_library",
         "//staging/src/kubevirt.io/client-go/kubecli:go_default_library",
@@ -54,6 +53,7 @@
         "//vendor/k8s.io/api/core/v1:go_default_library",
         "//vendor/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1:go_default_library",
         "//vendor/k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset:go_default_library",
+        "//vendor/k8s.io/apimachinery/pkg/api/errors:go_default_library",
         "//vendor/k8s.io/apimachinery/pkg/api/resource:go_default_library",
         "//vendor/k8s.io/apimachinery/pkg/apis/meta/v1:go_default_library",
         "//vendor/k8s.io/apimachinery/pkg/fields:go_default_library",
```

`ldd` output looks like this with it again
```bash
$ ldd _out/tests/tests.test     
        linux-vdso.so.1 (0x00007ffde7367000)
        libresolv.so.2 => /lib64/libresolv.so.2 (0x00007f7e78f21000)
        libc.so.6 => /lib64/libc.so.6 (0x00007f7e78d3f000)
        /lib64/ld-linux-x86-64.so.2 (0x00007f7e78f4e000)
```

Reply:
Thanks for spotting this. My IDE added the import automatically, and I haven't checked.